CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+> Please note that this project is released with a [Contributor Code of Conduct](#CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
```

Comment:
Sorry, this is my fault. Could you remove the `#` from the link, so that it points to the file, rather than trying to find an anchor on the current page?